uitests/plugins/org.jboss.tools.fuse.reddeer/src/org/jboss/tools/fuse/reddeer/editor/CamelDataFormatDialog.java

```diff
@@ -10,83 +10,83 @@
  ******************************************************************************/
 package org.jboss.tools.fuse.reddeer.editor;
 
+import java.util.ArrayList;
 import java.util.List;
 
-import org.eclipse.reddeer.common.wait.WaitUntil;
 import org.eclipse.reddeer.jface.wizard.WizardDialog;
-import org.eclipse.reddeer.swt.condition.ShellIsAvailable;
-import org.eclipse.reddeer.swt.impl.button.PushButton;
 import org.eclipse.reddeer.swt.impl.combo.LabeledCombo;
 import org.eclipse.reddeer.swt.impl.shell.DefaultShell;
 import org.eclipse.reddeer.swt.impl.text.LabeledText;
-import org.eclipse.reddeer.swt.impl.tree.DefaultTreeItem;
+import org.jboss.tools.fuse.reddeer.editor.ConfigurationsEditor.Element;
 
 /**
- * Manipulates with dialog (Wizard) for adding Global Camel Data Formats
+ * Dialog (wizard) for creating a new Camel Data Format
  * 
  * @author djelinek
  */
 public class CamelDataFormatDialog extends WizardDialog {
-	
-	public static final String TYPE = "Red Hat Fuse";
-	
-	public void activate() {
-		new WaitUntil(new ShellIsAvailable("Create a new Data Format..."));
-		new DefaultShell("Create a new Data Format...");		
+
+	public static final String TITLE = "Create a new Data Format...";
+
+	public CamelDataFormatDialog() {
+		super(TITLE);
+	}
+
+	public CamelDataFormatDialog activate() {
+		setShell(new DefaultShell(TITLE));
+		return this;
 	}
-	
+
 	public LabeledText getId() {
-		return new LabeledText("Id: *");
+		activate();
```

Comment:
Just fyi, it is not necessary to call activate() if you use the reference composite on the next line.